app/main.py

```diff
@@ -2,0 +1,24 @@
+import os
+
+
+def move_file(command: str) -> None:
+    com_ls = command.split()
+    path = com_ls[2]
```

Comment:
The variable name `path` could be more descriptive. Consider renaming it to `destination_path` to clarify its purpose.